charts/victoria-metrics-auth/templates/pvc.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
```

Comment:
maybe its not needed?

Reply:
Thanks @f41gh7, I will remove this file in the next commit.